api/v1beta1/types.go

```diff
@@ -243,6 +243,10 @@ type LoadBalancerSpec struct {
 	// FrontendIPsCount specifies the number of frontend IP addresses for the load balancer.
 	// +optional
 	FrontendIPsCount *int32 `json:"frontendIPsCount,omitempty"`
+	// BackendPool specifies the name of backend pool for the load balancer. If not set, it will
+	// be defaulted to the name of the load balancer name with "-outboundBackendPool" suffix.
+	// +optional
+	BackendPool BackendPool `json:"backendPool,omitempty"`
```

Comment:
Is there any reason not to just add a single property?:

```BackendPoolName *string `json:"backendPoolName,omitempty"```

Why add a `struct` type if the only property is the name?

Reply:
Adding a struct allows for future expansion of the configuration, if we later wanted to add some additional config, it would be preferable to have them all in a struct rather than `BackendPoolName`, `BackendPoolPort`, etc